### ISSUE TITLE
Demand graphs: Key Error when run from Toolbox

### ISSUE THREAD

**Issue by daren-thomas** [architecture-building-systems/CEAforArcGIS#309]
Reproduction steps:

using Zug case study, select the first three Variables to analyse:

![image](https://cloud.githubusercontent.com/assets/2969564/17887239/da43b664-6925-11e6-9870-66a30c93ee55.png)

Expected: Script to run through.

Buggy behavior: Results in an error message.

```
Executing: GraphsDemandTool C:\reference-case-zug\baseline Qww_tankloss_kWh;Trdata_C;QHf_kWh;mcphs_kWC
Start Time: Tue Aug 23 11:38:18 2016
Running script GraphsDemandTool...

Traceback (most recent call last):
  File "C:\Users\darthoma\Documents\GitHub\CEAforArcGIS\cea\GUI\toolbox.py", line 393, in execute
    cea.plots.graphs.graphs_demand(locator=locator, analysis_fields=analysis_fields, gv=gv)
  File "C:\Users\darthoma\Documents\GitHub\CEAforArcGIS\cea\plots\graphs.py", line 57, in graphs_demand
    total_demand = total_file[fields2]
  File "C:\Python27\ArcGIS10.4\lib\site-packages\pandas\core\frame.py", line 1785, in __getitem__
    return self._getitem_array(key)
  File "C:\Python27\ArcGIS10.4\lib\site-packages\pandas\core\frame.py", line 1829, in _getitem_array
    indexer = self.ix._convert_to_indexer(key, axis=1)
  File "C:\Python27\ArcGIS10.4\lib\site-packages\pandas\core\indexing.py", line 1103, in _convert_to_indexer
    raise KeyError('%s not in index' % objarr[mask])
KeyError: "[u'Trdata_MWhyr' u'mcphs_MWhyr'] not in index"

Failed to execute (GraphsDemandTool).
Failed at Tue Aug 23 11:38:18 2016 (Elapsed Time: 0.14 seconds)
```
Testing with other variables (["Ealf_kWh", "Qhsf_kWh", "Qwwf_kWh", "Qcsf_kWh"] as in the test-script) works as expected. So... some variables can be plotted and some can't. Why?

**Comment by daren-thomas** [comment]
@JIMENOFONSECA any idea? I'm not sure what the script does anymore, I'd have to look back into it. Maybe there is a mismatch between names in the Totals csv file and the per-building csv file?

**Comment by daren-thomas** [comment]
According to @gabriel-happle: 

> gabriel-happle [1:23 PM]  
@daren-thomas I might be the problem. I added columns to the csv files according to an old issue 'detailed electricity' (or similar). individual building reports have more columns added than the totals file.

Hm... What do you suggest we do? I can actually check to make sure we only use columns that are in both files...

**Comment by daren-thomas** [comment]
Fixed in PR #313 - waiting for merge

**Comment by daren-thomas** [comment]
@JIMENOFONSECA: Please test and merge.
@amraladdin: Can you do your testing magic and break this if possible?
